server/settings/_base.py

```diff
@@ -53,6 +53,11 @@ def initialize_config(cls):
             cls.SQLALCHEMY_DATABASE_URI = (sqlite_prefix +
                     os.path.abspath(cls.SQLALCHEMY_DATABASE_URI[len(sqlite_prefix) + 1:]))
 
+        sql_ca_cert = os.getenv('SQL_CA_CERT')
```

Comment:
Should this fall back to `BaltimoreCyberTrustRoot.crt.pem` since this is being included in the repo now?

Reply:
Good idea. Done and rebase the latest master.